pvp/common/message.py

```diff
@@ -64,12 +64,8 @@ def __init__(self, timestamp=None, loop_counter=None, breath_count=None, vals=ty
                         # otherwise just make one
                         self.timestamp = time.time()
                     else:
-                        raise e
-
-
-        # insist that we have all the rest of the vals
-        assert(all([value.name in kwargs.keys() for value in values.SENSOR.keys()]))
```

Comment:
why was this removed?

Reply:
I felt like the asserts should be removed in the deployed code